CHANGELOG.rst

```diff
@@ -36,6 +36,10 @@
   Thanks `@bagerard`_ for reporting (`#1503`_). Thanks to `@davehunt`_ and
   `@tomviner`_ for PR.
 
+* Whitelisted pytest warnings to show up warnings summary by default. Added a new
```

Comment:
"Whitelisted pytest warnings" seems confusing - it sounds like some warnings are on a whitelist.